test/fixtures/cluster-preload.js

```diff
@@ -1,3 +1,15 @@
+var assert = require('assert');
+
+// https://github.com/nodejs/io.js/issues/1803
+// this module is used as a preload module. It should have a parent with the
+// module search paths initialized from the current working directory
+assert.ok(module.parent);
+var expectedPaths = require('module')._nodeModulePaths(process.cwd());
+assert.ok(module.parent.paths.length === expectedPaths.length &&
```

Comment:
Are you trying to accomplish `assert.deepEqual(module.parent.paths, expectedPaths)` here?